nuxeo-features/nuxeo-automation/nuxeo-automation-core/src/main/java/org/nuxeo/ecm/automation/core/scripting/CoreFunctions.java

```diff
@@ -44,4 +44,10 @@ public String escapeHtml(Object obj) {
         return StringEscapeUtils.escapeHtml4(obj.toString());
     }
 
+    /**
+     * @since 11.1
+     */
```

Comment:
I know it's not always been done in the past, but please add a Javadoc to all public methods, even if it's just `Converts the {@link Date} object to a {@link DateWrapper}`.

Reply:
Done !